ospfd/ospf_vty.c

```diff
@@ -12120,6 +12120,10 @@ static int config_write_ospf_external_aggregator(struct vty *vty,
 			vty_out(vty, "\n");
 		}
 
+	if (ospf->aggr_delay_interval != OSPF_EXTL_AGGR_DEFAULT_DELAY)
+		vty_out(vty, " aggregation timer %u\n",
+			ospf->aggr_delay_interval);
+
```

Comment:
This config is printed from `ospf6_asbr_summary_config_write`

Reply:
Yes, but here is ospf ipv4.

Reply:
Right, sorry, my mistake.